numpy/linalg/linalg.py

```diff
@@ -226,6 +227,22 @@ def _assertNoEmpty2d(*arrays):
         if _isEmpty2d(a):
             raise LinAlgError("Arrays cannot be empty")
 
+def transpose(a):
+    """
+    Transpose each matrix in a stack of matrices.
```

Comment:
p.s. In the astropy case this is only used internally in cases where I *know* the inputs are arrays; if we exposed this publically in numpy, I think using the function makes more sense.